lib/Updater.php

```diff
@@ -435,23 +444,36 @@ private function getUpdateServerResponse() {
 	 *
 	 * @throws \Exception
 	 */
-	public function downloadUpdate() {
+	public function downloadUpdate()
+	{
 		$this->silentLog('[info] downloadUpdate()');
 
 		$response = $this->getUpdateServerResponse();
-		$storageLocation = $this->getDataDirectoryLocation() . '/updater-'.$this->getConfigOption('instanceid') . '/downloads/';
-		if(file_exists($storageLocation)) {
+		$storageLocation = $this->getDataDirectoryLocation() . '/updater-' . $this->getConfigOption('instanceid') . '/downloads/';
+		if (file_exists($storageLocation)) {
 			$this->silentLog('[info] storage location exists');
 			$this->recursiveDelete($storageLocation);
 		}
 		$state = mkdir($storageLocation, 0750, true);
-		if($state === false) {
+		if ($state === false) {
```

Comment:
The three whitespace changes then also need to be made to `index.php` because we have a copy of this there and a CI job checking that both are the same.